### PR TITLE
Fix Gaussian kernel issue

### LINKED ISSUE #272: [BUG] Changes to astropy.convolution.Gaussian2DKernel break tests
**System setup**
OS: [e.g] macOS v10.14.1
Python version: [e.g.] v3.6.7
Python environment (if any): [e.g.] conda v4.5.11

**Describe the bug**
ModOpt unit test `test_gaussian_kernel_none` is broken as the Astropy [Gaussian2DKernel](https://docs.astropy.org/en/v5.2/api/astropy.convolution.Gaussian2DKernel.html) method has been updated in v5.2 (see [changelog](https://docs.astropy.org/en/stable/changelog.html#id4))

**To Reproduce**
Run unit tests with Astropy installed.

**Are you planning to submit a Pull Request?**
- [X] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by sfarrens**
closes #272 

- Following [this Astropy issue](https://github.com/astropy/astropy/pull/13299), it is no longer possible to return an unnormalised Gaussian kernel. Therefore, I have removed this option from `gaussian_kernel` and the corresponding test.
- Following [this Scikit-Image issue](https://github.com/scikit-image/scikit-image/pull/6612), a maximum version of the package has been temporarily been set for the tests.

### CODE REVIEW

File: modopt/math/stats.py [hunk 1]
```diff
@@ -60,22 +62,22 @@ def gaussian_kernel(data_shape, sigma, norm='max'):
...
+    if norm == "max":
         return kernel / np.max(kernel)
 
-    elif norm == 'sum':
+    elif version.parse(astropy_version) < version.parse("5.2"):
         return kernel / np.sum(kernel)
 
-    elif norm == 'none':
+    else:
         return kernel
```
Comment (paquiteau) [inline modopt/math/stats.py:81]: The documentation should be updated to reflect this new behavior

**Review by paquiteau — CHANGES_REQUESTED** [review]
Thanks for the PR,

only a docstring update is missing, otherwise, I am good with it.

File: modopt/math/stats.py (hunk 1 above)
Comment (sfarrens) [reply modopt/math/stats.py:80]: Yep, thanks @paquiteau, sorted!

**Comment by sfarrens** [comment]
Hey @paquiteau, if you are happy with this can you click on the "Approve" button? 🙂

**Review by paquiteau — APPROVED** [review]

**Pull request merged**